rtmp-client/src/main/cpp/librtmp/rtmp.h

```diff
@@ -176,7 +176,8 @@ extern "C"
     int swfAge;
 
     int protocol;
-    int timeout;		/* connection timeout in seconds */
+    int timeout;                /* connection timeout in milliseconds */
```

Comment:
Can this be renamed to `receiveTimeoutInMs`, for consistency?

Note that you can remove the comment on this line and the one below if so, because the names are then clear enough to be self documenting.

Reply:
Done